packages/tfm-ror52/tfm-ror52.spec

```diff
@@ -86,11 +87,11 @@ help2man -N --section 7 ./h2m_help -o %{scl_name}.7
 %scl_install
 
 cat >> %{buildroot}%{_scl_scripts}/enable << EOF
-export PATH=%{_bindir}\${PATH:+:\${PATH}}
-export LD_LIBRARY_PATH=%{_libdir}\${LD_LIBRARY_PATH:+:\${LD_LIBRARY_PATH}}
-export MANPATH=%{_mandir}:\${MANPATH}
-export PKG_CONFIG_PATH=%{_libdir}/pkgconfig\${PKG_CONFIG_PATH:+:\${PKG_CONFIG_PATH}}
-export GEM_PATH=\${GEM_PATH:=%{gem_dir}:\`scl enable %{scl_ruby} -- ruby -e "print Gem.path.join(':')"\`}
+export PATH="%{_bindir}:%{_sbindir}\${PATH:+:\${PATH}}"
+export LD_LIBRARY_PATH="%{_libdir}\${LD_LIBRARY_PATH:+:\${LD_LIBRARY_PATH}}"
+export MANPATH="%{_mandir}:\${MANPATH:-}"
+export PKG_CONFIG_PATH="%{_libdir}/pkgconfig\${PKG_CONFIG_PATH:+:\${PKG_CONFIG_PATH}}"
+export GEM_PATH="\${GEM_PATH:=%{gem_dir}:\`scl enable %{scl_ruby} -- ruby -e "print Gem.path.join(':')"\`}"
 
 . scl_source enable %{scl_ruby}
```

Comment:
We already did this in ror51 but should we first enable the Ruby SCL and then ours so the path precedence is tfm-ror52 -> rh-ruby25 -> system paths instead of rh-ruby25 -> tfm-ror52 -> system paths.

Reply:
This *is* the tfm-ror52 enable script so we can't enable itself from inside itself :)

Reply:
Sorry, was a bit unclear. I meant that we should source the scl_ruby and then export variables. Right now we first export and then source.

Reply:
Maybe that's better? I can try it, but all documentation and existing SCLs do it this way.

Reply:
Let's leave it this way then.